### PR TITLE
scale-test: disable kube-proxy and configure KPR

### PR DESCRIPTION
**Pull request by thorn3r**
Update the create-cluster action to latest to enable deploying a cluster without kube-proxy, and configure Cilium with KubeProxyReplacement.

### CODE REVIEW

**Review by viktor-kurchenko — APPROVED** [review]

File: .github/workflows/scale-test-100-gce.yaml [hunk 1]
```diff
@@ -226,10 +229,24 @@ jobs:
...
+          --set k8sServicePort=443"
+          echo cilium_install_apiserver=${CILIUM_INSTALL_APISERVER} >> $GITHUB_OUTPUT
+
       - name: Install Cilium
         run: |
-          cilium install --dry-run-helm-values ${{ steps.vars.outputs.cilium_install_defaults }}
-          cilium install ${{ steps.vars.outputs.cilium_install_defaults }}
+          cilium install --dry-run-helm-values \
+          ${{ steps.vars.outputs.cilium_install_defaults }} \
+          ${{ steps.apiserver-vars.outputs.cilium_install_apiserver }}
```
Comment (giorio94) [inline .github/workflows/scale-test-100-gce.yaml]: Drive-by comment, an alternative (and slightly simpler) approach used for instance in the scale clustermesh workflow is:

```
    --set k8sServiceHost=api.internal.${{ steps.vars.outputs.cluster_name }} \
    --set k8sServicePort=443 \
```

That mimics how kops configures kube-proxy for instance.

Comment (learnitall) [reply .github/workflows/scale-test-100-gce.yaml]: Oh cool! Could we try this @thorn3r?

Comment (thorn3r) [reply .github/workflows/scale-test-100-gce.yaml]: Nice! This is much simpler, updated :+1:

**Comment by thorn3r** [comment]
Fixed a merge conflict and the placement of the args (they were in the SHA conditional block)

**Review by learnitall — APPROVED** [review]
LGTM!

**Review by giorio94 — APPROVED** [review]
